acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/BesuNodeConfigurationBuilder.java

```diff
@@ -234,6 +235,12 @@ public BesuNodeConfigurationBuilder staticNodes(final List<String> staticNodes)
     return this;
   }
 
+  public BesuNodeConfigurationBuilder privacyParameters(
+      final Optional<PrivacyParameters> privacyParameters) {
```

Comment:
nit:_personally I'd rather see someone pass in a concrete privacyParameters, then this function can do a "Optional.ofNullable" - the person injecting the data doesn't (necessarily) want to have to construct up an Optional to make this work (unless this is invoked from a simple pass-through function)